test/app/src/main.ts

```diff
@@ -12,17 +12,11 @@ declare const CONFIG: {
   OKTA_TESTING_DISABLEHTTPSCHECK: string;
 }
 
-// To perform end-to-end PKCE flow we must be configured on both ends: when the login is initiated, and on the callback
-// The login page is loaded with a query param. This will select a unique callback url
-// On the callback load we detect PKCE by inspecting the pathname
-const url = new URL(window.location.href)
-const pkce = !!url.searchParams.get('pkce') || url.pathname.indexOf('pkce/callback') >= 0
-const redirectUri = window.location.origin + (pkce ? '/pkce/callback' : '/implicit/callback')
+const redirectUri = window.location.origin + '/login/callback'
```

Comment:
I'll defer to Aaron on this one, since he wrote the two-option callback in the first place to ensure we aren't losing any test coverage.

Reply:
This logic is from the early days before we had good test coverage on auth-js.